addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_image.py

```diff
@@ -93,20 +95,14 @@ def __gather_extras(sockets, export_settings):
 
 
 def __gather_mime_type(sockets, export_image, export_settings):
-    # force png if Alpha contained so we can export alpha
```

Comment:
If the user specifically sets their export mode to jpeg, I think things should always export as a jpeg. When set to auto the new `preffered_mime_type()` check would naturally output as a png unless the input image was already a jpg